qcodes/instrument_drivers/tektronix/AWG5014.py

```diff
@@ -341,328 +345,426 @@ def __init__(self, name, setup_folder, address, reset=False,
 
     # Functions
     def get_all(self, update=True):
```

Comment:
Why not remove this (or at least add a depreciation warning) and let people use snapshot(update=True)